apps/meteor/app/livechat/imports/server/rest/sms.ts

```diff
@@ -128,6 +128,7 @@ API.v1.addRoute('livechat/sms-incoming/:service', {
 			source: {
 				type: OmnichannelSourceType.SMS,
 				alias: service,
+				destination: sms.to,
```

Comment:
it doesn't look like that types are being enforced here (you can send anything as roomInfo to the createRoom function). Can you check if you can add the proper types to createRoom without having to change anything else? Otherwise just set the type of this roomInfo object.